integrations/acquisition/covidcast/test_delete_batch.py

```diff
@@ -25,37 +25,22 @@ class DeleteBatch(unittest.TestCase):
     def setUp(self):
         """Perform per-test setup."""
 
-        # connect to the `epidata` database and clear the `covidcast` table
-        cnx = mysql.connector.connect(
-            user='user',
-            password='pass',
-            host='delphi_database_epidata',
-            database='epidata')
-        cur = cnx.cursor()
-        cur.execute('truncate table covidcast')
-        cnx.commit()
-        cur.close()
-
-        # make connection and cursor available to test cases
-        self.cnx = cnx
-        self.cur = cnx.cursor()
-
         # use the local instance of the epidata database
         secrets.db.host = 'delphi_database_epidata'
         secrets.db.epi = ('user', 'pass')
 
-        # use the local instance of the Epidata API
-        Epidata.BASE_URL = 'http://delphi_web_epidata/epidata/api.php'
-
         # will use secrets as set above
-        from delphi.epidata.acquisition.covidcast.database import Database
-        self.database = Database()
-        self.database.connect()
+        self._db = Database()
+        self._db.connect()
+
+        for table in "signal_load  signal_latest  signal_history  geo_dim  signal_dim".split():
+            self._db._cursor.execute(f"TRUNCATE TABLE {table}")
+
 
     def tearDown(self):
         """Perform per-test teardown."""
-        self.cur.close()
-        self.cnx.close()
+        self._db.disconnect(False)
+        del self._db
```

Comment:
no...  but it doesnt hurt?  if there is any GC that happens when its reference is lost, it happens on `tearDown()` instead of `setUp()` when a new `Database` object is created.  plus it returns the `DeleteBatch` (test) object to a more pristine state that better resembles a newly created object.

Reply:
it hurts readability if it's inconsistent. we don't delete this property anywhere else, so unless this test creates unique circumstances, we should probably not do it here.

Reply:
this is a recommendation for inclusion in the test refactor -- should be moved there